### ISSUE TITLE
Should add nix target for building .pdf (and .tex?) version of the tech report

### ISSUE THREAD

**Issue by apoelstra** [BlockstreamResearch/simplicity#141]

**Comment by roconnor-blockstream** [comment]
Related: https://savannah.gnu.org/bugs/?32171

**Comment by roconnor-blockstream** [comment]
Let's keep this issue open even after #145.  When a new version of TeXmacs is released we can replace `xvfb-run` with just a simple `--headless` argument to TeXmacs.

**Comment by roconnor-blockstream** [comment]
--headless will probably be available in Texmacs 2.1.4, which was merged into [nixpkgs on 2024-09-08](https://github.com/NixOS/nixpkgs/pull/340621)

We should wait for this to get into nixos-stable
